GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Ariane.cfg

```diff
@@ -268,34 +270,29 @@
     @manufacturer = Airbus Defence and Space
     @description = The second stage of the Ariane 5 launch vehicle family (ESC-A variant). Contains guidance capability.
 
-    @mass = 4.375
+    @mass = 4.45
     @crashTolerance = 14
     @breakingForce = 250
     @breakingTorque = 250
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
-    @MODULE[ModuleRCS]
+    //  56 N Gas Hydrogen (GH2) attitude control thrusters for each axis (using liquid hydrogen directly for now).
```

Comment:
Does it really? Very interesting. As long as you make the proper conversion between LH2 & GH2 densities and masses, usingn LH2 should be fine.